main.go

```diff
@@ -571,6 +591,81 @@ func updateScreenshotNames(testLogsDir string) error {
 	return nil
 }
 
+func updateOldSummaryTypeScreenshotName(testItem map[string]interface{}, testLogsDir, uuid string, startTime time.Time) (string, bool, error) {
+	var origScreenshotPth string
+
+	for _, ext := range []string{"png", "jpg"} {
+		origScreenshotPth = filepath.Join(testLogsDir, "Attachments", fmt.Sprintf("Screenshot_%s.%s", uuid, ext))
+		var newScreenshotPth string
```

Comment:
We don't need to search for .png and .jpg in the **new** version of _TestSummaries.plist because it has a `Filename` field which contains the extension already.